website/docs/feature-flag-tutorials/python/implementing-feature-flags.md

```diff
@@ -0,0 +1,346 @@
+
+
+![Image of a feature flag form](/img/python-tutorial-feature-flag-form.png)
+
+
+Your new feature flag has been created and is ready to be used. Enable the flag for your development environment, which makes it accessible for use in the Python app we will clone into your local environment.
+
+
+![Image of the enabled Python flag in development environment](/img/python-tutorial-enabled-flag.png)
+
+Next, generate an API token to authenticate calls made to Unleash servers from your project. This API token will eventually be pulled into a configuration object within your Python application to toggle features. 
+
+> **Note** We require an API token as part of your flag configuration to ensure that only applications with the correct authentication can access your feature flags in Unleash. API tokens are created by users with API management access and thus controls how they can be distributed to applications that need it, and by whom.
+
+From your project view on the platform, go to "Project Settings" and then "API Access".
+
+Select the ‘New API token’ button.
+
+
+![Image of the API token button in API Access view](/img/python-tutorial-create-api-token.png)
+
+
+Name the API token and select the “Server-side SDK” token type, since we’ll be doing our flag evaluation on the server using the Python SDK. You can read more about [Unleash API tokens in our documentation](https://docs.getunleash.io/reference/api-tokens-and-client-keys#client-tokens).
+
+The token should have access to the “development” environment, as shown in the platform screenshot below.
+
+
+![Image of the API token creation form](/img/python-tutorial-create-api-token-form.png)
+
+
+The API token you generated can be managed in the API Access view in your project settings. It will become handy in Step 4.
+
+
+## 4. Add Unleash to a Python app
+
+In this section, you will clone an open-source Python application called [Flask Surveys Containers app](https://github.com/pamelafox/flask-surveys-container-app), which we are using to model a service that provides routing, serves HTML pages, and performs actions against a database. This app uses Flask, SQLAlchemy, and a PostgreSQL database.
+
+Use this command to clone the repository via your Terminal:
+
+```
+git clone git@github.com:pamelafox/flask-surveys-container-app.git
+```
+
+Next, navigate into your repository directory and create a `.env` file.
+
+Copy this code snippet into your `.env` file:
+
+
+```py
+FLASK_DEBUG=True
+DBHOST=db
+DBNAME=postgres
+DBUSER=app_user
+DBPASS=app_password
+```
+
+
+Next, install the Python SDK. Open your repository in a code editor and navigate to `requirements.txt` inside the `src` folder. Reference the Python SDK for installation.
+
+
+```py
+UnleashClient==5.11.1
+```
+
+
+In `src/backend/__init__.py`, import UnleashClient:
+
+
+```py
+from UnleashClient import UnleashClient
+```
+
+
+In the same file, call the Unleash client for initialization when the app runs with this code snippet:
+
+
+```py
```

Comment:
A few things about this snippet:

1) The url you've chosen here won't work on Linux machines - this is kinda a little hack to bridge two compose runs
2) The example has mixed tabs and spaces, which makes this horrid to copy paste, since a Python interpreter will panic